src/Uno.UI.Toolkit/ElevatedView.cs

```diff
@@ -133,11 +133,7 @@ public new Brush Background
 			set => SetValue(BackgroundProperty, value);
 		}
 
-		public
-#if __SKIA__
-			new
-#endif
-			static DependencyProperty CornerRadiusProperty { get ; } = DependencyProperty.Register(
+		public new static DependencyProperty CornerRadiusProperty { get ; } = DependencyProperty.Register(
```

Comment:
It looks like we should be able to remove the property completely and use the one from `Control`? That's a binary breaking change though and should be made separately, probably as part of the breaking change rollup.

Thoughts @jeromelaban ?

Reply:
> Indeed we should be able to.

Follow-up PR is good or do I need to comment in the breaking change rollup issue?

Reply:
No, it's a breaking change so we'll bundle it for the next major release.